Content.Server/Shuttles/Systems/ShuttleConsoleSystem.cs

```diff
@@ -271,9 +271,11 @@ private void UpdateState(EntityUid consoleUid, List<DockingInterfaceState>? dock
             ftlTime = _timing.CurTime + TimeSpan.FromSeconds(shuttleFtl.Accumulator);
         }
 
-        // Mass too large
+        // Is Mass of grid too large?
+        // OR is intended to be a Spaceship
+        // The 1000 mass limit exists as abstract divide between which grid "is a shuttle" and which "is a station"
         if (entity != null && shuttleGridUid != null &&
-            (!TryComp<PhysicsComponent>(shuttleGridUid, out var shuttleBody) || shuttleBody.Mass < 1000f))
+            (!TryComp<PhysicsComponent>(shuttleGridUid, out var shuttleBody) || (HasComp<FTLWhitelistComponent>(shuttleGridUid) || shuttleBody.Mass < 1000f)))
```

Comment:
```suggestion
            (!TryComp<PhysicsComponent>(shuttleGridUid, out var shuttleBody) || HasComp<FTLWhitelistComponent>(shuttleGridUid) || shuttleBody.Mass < 1000f))
```

The additional parenthetical is superfluous by the nature of how these conditions are evaluated. You can look up order of operations to learn more.